### PR TITLE
Add ruleset for asrock.com

### PR DESCRIPTION
**Pull request by phuzion**
Adds asrock.com, www.asrock.com and events.asrock.com, per request in #13374

### CODE REVIEW

File: src/chrome/content/rules/Asrock.com.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!--
+        Nonfunctional hosts in *.asrock.com:
```
Comment (J0WI) [inline src/chrome/content/rules/Asrock.com.xml]: Please add:
```
cn.asrock.com
download.asrock.com
europe.asrock.com
forum.asrock.com
oc.asrock.com
tw.asrock.com

**Comment by Bisaloo** [comment]
@phuzion, are you still interested in working on this ruleset?

**Comment by Bisaloo** [comment]
Can you please restore your branch and check the box "Allow edits from maintainers"? It's probably the easiest way for one of us to take over.

**Comment by phuzion** [comment]
Done.

**Comment by Bisaloo** [comment]
Thank you! @J0WI, do you want to continue here or should I do it?

**Comment by J0WI** [comment]
I've added the missing hosts from https://github.com/EFForg/https-everywhere/pull/13464#discussion_r148935825. Could you have an additional look my changes?

File: src/chrome/content/rules/Asrock.com.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+			- tw.asrock.com (r)
+
+        h: http redirect
+        m: certificate mismatch
+        r: connection refused
+        s: self-signed certificate
+        t: timeout on https
+-->
+<ruleset name="Asrock.com">
+        <target host="asrock.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Asrock.com.xml]: Fix indents to use tabs instead of spaces.

**Pull request merged**